core/chains/evm/config/v2/config_ocr2.go

```diff
@@ -0,0 +1,23 @@
+package v2
+
+import "github.com/smartcontractkit/chainlink/v2/core/chains/evm/config"
+
+type ocr2Config struct {
+	c OCR2
+}
+
+type ocr2Automation struct {
+	c Automation
+}
+
+func (o ocr2Automation) GasLimit() uint32 {
+	return *o.c.GasLimit
+}
```

Comment:
Order and pointer receiver?
```suggestion
type ocr2Automation struct {
	c Automation
}

func (o *ocr2Automation) GasLimit() uint32 {
	return *o.c.GasLimit
}

type ocr2Config struct {
	c OCR2
}
```

Reply:
Fixed in [`84f922f` (#9606)](https://github.com/smartcontractkit/chainlink/pull/9606/commits/84f922fd3885d4a8ff29121bba8cb0e5597a68ca)